theme/highlight.js

```diff
@@ -0,0 +1,1328 @@
+/*
+  Highlight.js 10.7.2 (00233d63)
+  License: BSD-3-Clause
+  Copyright (c) 2006-2021, Ivan Sagalaev
+*/
+var hljs=function(){"use strict";function e(t){
```

Comment:
Does the theme have to be included here?

Reply:
You mean this Javascript file? Yes, because mdBook by default doesn't support protobuf syntax highlighting: https://rust-lang.github.io/mdBook/format/theme/syntax-highlighting.html#supported-languages

Reply:
Yeah exactly, I would think that one can configure this instead of including the file. But I haven't tried myself.

Reply:
Unfortunately, I don't think you can 🤷.